usaspending_api/references/v2/views/city.py

```diff
@@ -95,29 +92,58 @@ def create_elasticsearch_query(return_fields, scope, search_text, country, state
     return query
 
 
-def create_es_search(method, scope, search_text, country=None, state=None):
+def create_es_search(scope, search_text, country=None, state=None, term_level_query_method="wildcard"):
     """
         Providing the parameters, create a value query sub-string for elasticsearch
 
-        IF there is a need to perform a fuzzy search, there might be a need to set
-            ["query_string"]["fuzzy_prefix_length"] to a value like 1
+        Args:
+            scope: which city field was chosen for searching `pop` (place of performance) or `recipient_location`
+            search_text: the text the user is typing in and sent to the backend
+            country: optional country selected by user
+            state: optional state selected by user
+            term_level_query_method: Supports `wildcard` or `fuzzy`. Defaults to wildcard
     """
-    method_char = "~" if method == "fuzzy" else "*"
+    method_char = "" if term_level_query_method == "fuzzy" else "*"
```

Comment:
Removing the tilde changed how the fuzzy match behaves.  I'm not sure I understand the difference.  Both seem to be returning fuzzy matches, but with the tilde we don't get anything for "LOND" whereas we do without.  I think I like the sans tilde results better.

Either way, the fuzzy matching is not being used anywhere.  Is it worth keeping?

Reply:
Done. Removed fuzzy. It was cool but extraneous and not SOLID